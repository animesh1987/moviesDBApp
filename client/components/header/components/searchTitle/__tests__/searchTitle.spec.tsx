import * as React from 'react';
import { create } from 'react-test-renderer';

import { SearchTitle } from '../index';

describe('Search Title Test', () => {

  it('renders correctly', () => {
    const component = create(<SearchTitle />)
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});