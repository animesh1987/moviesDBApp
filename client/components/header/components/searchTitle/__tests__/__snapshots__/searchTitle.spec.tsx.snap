// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Title Test renders correctly 1`] = `
<div
  className="search-title"
>
  Find your movie
</div>
`;
